warehouse/admin/views/verdicts.py

```diff
@@ -61,11 +61,43 @@ def get_verdict(request):
     verdict = request.db.query(MalwareVerdict).get(request.matchdict["verdict_id"])
 
     if verdict:
-        return {"verdict": verdict}
+        return {
+            "verdict": verdict,
+            "classifications": list(VerdictClassification.__members__.keys()),
+        }
 
     raise HTTPNotFound
 
 
+@view_config(
+    route_name="admin.verdicts.review",
+    permission="admin",
```

Comment:
permission="moderator" is probably good here so that we can easily add additional volunteers to review.

Reply:
with that in mind, it might make sense to change the name from `administrator_verdict` to `reviewer_verdict`